dom.go

```diff
@@ -2072,6 +2194,11 @@ func (ctx *CanvasRenderingContext2D) Restore() {
 	ctx.Call("restore")
 }
 
+//Hit regions
+//addHitRegion
+//removeHitRegion
+//clearHitRegions
```

Comment:
Yeah i meant to put a TODO list there, i was using comments like this to note down wich functions i had to implement and where they had to go.

The only reason i didn't implement these is that those require the https://developer.mozilla.org/en-US/docs/Web/API/Path2D

If you think we need those i could implement those for another pull request.